Xamarin.Essentials/Sms/Sms.shared.cs

```diff
@@ -11,25 +13,23 @@ public static Task ComposeAsync(SmsMessage message)
         {
             if (!IsComposeSupported)
                 throw new FeatureNotSupportedException();
-
-            return PlatformComposeAsync(message);
+            var filteredMessage = new SmsMessage(
+                message.Body,
+                message.Recipients.Where(x => !string.IsNullOrEmpty(x)).ToArray());
+            return PlatformComposeAsync(filteredMessage);
         }
     }
 
     public class SmsMessage
     {
-        public SmsMessage()
-        {
-        }
-
-        public SmsMessage(string body, string recipient)
+        public SmsMessage(string body, string[] recipients)
         {
             Body = body;
-            Recipient = recipient;
+            Recipients = recipients;
```

Comment:
could/should we filter here in stead? Is that too weird?

Should it be IsNullOrWhitespace?

Reply:
I guess we could do that if we document it. Otherwise confusion could arise. NullOrWhitespace is indeed superior. Once it is decided where to filter I will use this instead

Reply:
Yeah, we could say that we will filter out any blank phone numbers.... i mean they aren't valid. If you set no numbers then I know it will atleast bring it up with a blank number....

Reply:
That does sound reasonable. Ill do that then

Reply:
Done

Reply:
:) just realized i had a huge typo and bad language in my comment. Whoops, sorry about that <3

Reply:
I did not even notice :)